Objects/complexobject.c

```diff
@@ -131,11 +131,8 @@ _Py_c_pow(Py_complex a, Py_complex b)
 {
     Py_complex r;
     double vabs,len,at,phase;
-    if (b.real == 0. && b.imag == 0.) {
-        r.real = 1.;
-        r.imag = 0.;
-    }
-    else if (a.real == 0. && a.imag == 0.) {
+    assert(!(b.real == 0. && b.imag == 0.));
```

Comment:
Please could you a comment explaining why we expect `b` to be nonzero?

Reply:
Oops, actually we could expect any garbage here.  I forgot that ``_Py_c_pow()`` and friends actually are public functions.

I'll revert this and add some trivial tests for these low-level functions to the Lib/test/test_capi/test_complex.py.

Reply:
Urgh. Public/private status is kinda messy right now. I agree that the safest course is to keep the branch, though.

Reply:
Huh, I'll just revert this.   #106339 makes them inaccessible from ``<Python.h>``.  This is a bug.

Reply:
FYI: #112019 - to track possible ``_Py_c_pow``, etc removal